app/api/bookyt/entities/base.rb

```diff
@@ -0,0 +1,10 @@
+module Bookyt
+  module Entities
+    class Base < Grape::Entity
+      format_with(:iso_timestamp) do |dt|
+        dt ? dt.iso8601 : nil
+      end
+      format_with(:float, &:to_f)
```

Comment:
I'm using BigDecimal instead of Float in the whole code base because of subtle rounding issues. I do not think this is an issue here. But I feel a bit of unease with float when dealing with financial data.